sam-styles/packages/branding/elements/_tags.scss

```diff
@@ -401,4 +401,46 @@
     line-height: 1.5;
     margin-left: 0.75rem;
     text-decoration: none;
-  }
\ No newline at end of file
+  }
+
+  .sds-tag-right-align {
+    position: absolute;
+    top: 12px;
+    right: 12px;
+    z-index: 10;
+  }
+  
+  .sds-status-tag {
+    @include u-font("sans", "3xs");
+    @include u-text("normal");
+    padding: 6px 10px !important;
+    border-radius: 99rem;
+    display: inline-flex; 
+    align-items: center;
+    justify-content: center;
+    border: 1px solid transparent;
+  
+    &--info {
+      @include u-bg("info-lighter");
+      @include u-border("info-light");
+    }
+  
+    &--error {
+      background-color: #fdecea; 
+      @include u-border("error-lighter");
+    }
+  
+    &--accent-cool {
+      @include u-bg("accent-cool-lighter");
+      @include u-border("accent-cool");
+    }
+    &--locked {
```

Comment:
Can you include some spacing between the icon and text to match the mockup?

Reply:
done

Reply:
Looks good